fullscreen.bs

```diff
@@ -270,6 +270,9 @@ when invoked, must run these steps:
    <li><p>This algorithm is <a>allowed to request fullscreen</a>.
   </ul>
 
+ <li><p>If <var>error</var> is false, then <a>consume user activation</a> for
```

Comment:
FYI: This should now point to the newly added definition of [consumption](https://html.spec.whatwg.org/multipage/interaction.html#consume-user-activation).  Not sure why the PR preview doesn't show a link.

Reply:
Looks like the preview is correctly pointing to the HTML spec link now.